articles/connections/passwordless/index.md

```diff
@@ -26,13 +26,15 @@ See the following tutorials for a step-by-step guide on how to implement passwor
 
 ### Passwordless on iOS
 
+ - [Authenticate users with Universal Login](/connections/passwordless/native-passwordless-universal)
```

Comment:
See whether it's universal login or Universal Login and update the PR accordingly (I've seen both)

Reply:
I'm asking about this - do you mind if I just standardize this in a separate PR? There are now multiple PRs + existing pages using this term, probably using it both ways. I'll get an answer and pull an Eric :D

Reply:
`pull an Eric` 😆

Reply:
No, I don't mind. Just make sure this doesn't slip through the cracks and when you do get an answer and send a PR, please make also sure that everyone on the team knows which one to use. Also add the correct one here https://github.com/auth0/docs/blob/master/WORDS.md (and add a note on the linting confluence page to enforce a rule about it)